public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -1,4 +1,4 @@
-function AdminGSVLabelView(admin) {
+function AdminGSVLabelView(admin, source) {
     var self = {};
     self.admin = admin;
```

Comment:
Let's just save the source parameter like we do with the admin parameter here. Then we don't have to pass it as a parameter to the `_validateLabel()` function and can use it directly in there.